Dockerfile

```diff
@@ -1,23 +1,23 @@
-ARG ORTOOLS_VERSION=${ORTOOLS_VERSION}
+ARG ORTOOLS_URL=${ORTOOLS_URL}
 ARG REGISTRY=${REGISTRY:-registry.mapotempo.com/}
 
-# Install ORTools
-FROM ${REGISTRY}mapotempo/ortools:${ORTOOLS_VERSION} as ortools
-
+# Final image
 # Build wrapper
+FROM debian:latest
+ARG ORTOOLS_URL
```

Comment:
This line looks like extra :thinking: is it needed?

Reply:
The variables defined before a FROM are default, but unknown if not called again in the current context